common/src/dodeca.rs

```diff
@@ -124,6 +126,14 @@ impl Vertex {
         adjacent_vertices()[self as usize]
     }
 
+    /// Chunk permutations adjacent to this vertex, opposite the sides in canonical order.
+    /// These permutations convert coordinates relative to a reflected version of the current
+    /// chunk to coordinates relative to the adjacent chunk in its canonical orientation.
```

Comment:
This is pretty opaque. Why are the coordinates initially reflected wrt. the local chunk, and in which way? Is a "chunk permutation" different from an axis permutation?

Reply:
I'll need to think a bit more about the phrasing of this. I'll also need to avoid the phrase "chunk permutation" because that makes no sense. I think I got that phrase by find/replacing orientation to permutation.

There is only one way to reflect a chunk and reach another chunk in the same dodeca, and that is to reflect it about the surface that they share. I'll probably want to mention/describe that surface in these docs, as it's pretty confusing.

As for "opposite the sides in canonical order", that also feels somewhat vague. I'll revisit all of this documentation before merging to make it less opaque.

Reply:
@Ralith, I believe I figured out a clearer way to word this documentation, but please double-check that it makes sense. I'll hold off on merging until then.